apps/sap-commerce-cloud/frontend/src/components/Dialog/Dialog.tsx

```diff
@@ -137,15 +127,10 @@ const Dialog: React.FC<DialogProps> = ({ sdk }) => {
         <GridItem>
           <TextInput
             type="text"
-            placeholder={'Search Term...'}
+            placeholder={'Type to search products'}
```

Comment:
I thought to change this placeholder just as a tiny thing to help make it a little more clear that the way to search is to type in a term. Let me know if there's a better convention here.